src/pages/email.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import { Button, Container, Input, Text, Textarea } from '@chakra-ui/react';
+
+const Email = () => {
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState<string>('');
+  const [emailText, setEmailText] = useState('');
+  const [emailRecipients, setEmailRecipients] = useState([]);
+
+  const handleEmailSend = async () => {
+    setLoading(true);
+    setMessage('');
+    try {
+      const response = await fetch('/api/sendEmail', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          to: emailRecipients,
+          subject: 'Hello World',
+          text: emailText,
+        }),
+      });
+
+      if (!response.ok) {
+        const errorData = await response.json();
+        throw new Error(errorData.details || 'Failed to send email');
+      }
+
+      const data = await response.json();
+      setMessage(data.message);
+    } catch (error: any) {
+      setMessage(`Error sending email: ${error.message}`);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const handleEmailTextChange = (e) => {
+    setEmailText(e.target.value);
+  };
+
+  const handleEmailRecipientsChange = (e) => {
+    const recipientsString = e.target.value;
+    setEmailRecipients(recipientsString.split(','));
+  };
+
+  return (
+    <Container>
+      <Input
+        placeholder="Add recipients here"
```

Comment:
Nothing to address here. For my knowledge's sake: I would usually expect:
```react
<Input placeholder="random placeholder" value={value}>
```
but we were able to do without this. How does this happen?
Does React not always need this `value={stateVar}` thing?
Would appreciate any feedback/links/something to read up

Reply:
I think for the Input element in HTML it just knows to store whatever you type in